todo_app/test/integration_tests/app_test.py

```diff
@@ -1,17 +1,18 @@
 import os
 import pytest
 from dotenv import load_dotenv, find_dotenv
-from todo_app import app
+from todo_app import todo_app
 import requests
 
 @pytest.fixture
 def client():
     # Use our test integration config instead of the 'real' version
-    file_path = find_dotenv('.env.test')
-    load_dotenv(file_path, override=True)
+    if (os.getenv('ENVIRONMENT') != 'docker'):
+        file_path = find_dotenv('.env.test')
+        load_dotenv(file_path, override=True)
```

Comment:
Minor, but I slightly prefer a try/fail pattern for this than checking the environment - then we can skip the `.env.docker` file and just having a single `.env.test` file for every environment
```
    try:
        file_path = find_dotenv('.env.test')
        load_dotenv(file_path, override=True)
    except OSError:
        print('Failed to load dotenv')
```